### ISSUE TITLE
FallbackInputHandler executing though other GameObject consumes the event 

### ISSUE THREAD

**Issue by joherb** [Microsoft/MixedRealityToolkit-Unity#1558]
## Overview
A FallbackInputHandler's OnInputClicked() is executing thogh other focussed GameObject consumes the event.
## Expected Behavior
Any input handlers on the game object are given input events when no modal or focused objects consume the event.
## Actual Behavior
Event gets handled by focused object and fallbackInputStack.
## Steps to reproduce
Create a scene with HololensCamera, DefaultCursor and InputManager. Add two cubes with scripts implementing IInputClickHandler. Push one of them to fallbackInputStack. 
## Unity Editor Version
Unity 2017.1.2f1 (64-bit)
## Mixed Reality Toolkit Release Version
HoloToolkit-Unity-v1.2017.2.0

**Comment by StephenHodgson** [comment]
Did you use `eventData.Use();`?

**Comment by joherb** [comment]
Yes I do, but the somehow the one from fallbackstack gets triggert bevor the focused one. Updated to Unity 2017 2.1f1 and HoloToolkit-Unity-2017.2.1.0.

**Comment by joherb** [comment]
Ok. Just took a look into my test setup like described in "Steps to reproduce". Forgot to call `eventData.Use();` and it does the trick! So I have to figure out where what's going wrong in my main project... Thanks!

**Comment by joherb** [comment]
For the record: I found the problem in my project. The GameObject with fallback handler script also uses HandDraggable.cs fom MRTK. I changed `InputManager.Instance.PushModalInputHandler(gameObject);` to `InputManager.Instance.PushModalInputHandler(HostTransform.gameObject);` in there what is ok for me because I have set HostTransform to another GameObject and enable/disable HandDraggable somewhere else.
Now everything is working like expected and this should be closed.
